fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -158,7 +163,8 @@ class ApplyMetadataMutationsImpl {
 
 private:
 	void writeMutation(const MutationRef& m) {
-		if (forResolver || !SERVER_KNOBS->ENABLE_TLOG_ENCRYPTION) {
+		if (forResolver ||
+		    !isEncryptionEnabled(EncryptOperationType::TLOG_ENCRYPTION, dbInfo->get().client.isEncryptionEnabled)) {
```

Comment:
An alternate would be to pass `ClientDBInfo` to `isEncryptionEnabled` routine and let it abstract out details. One advantage would be: if ClientDBInfo internal flag usage is updated (which is going to happen as we know), then call sites need not be updated (several) and the PR to make check again complaint would be simple and minimal. Thoughts?

Reply:
Yeah good point, will change that